cupy/cuda/runtime.pxd

```diff
@@ -67,6 +67,106 @@ cpdef enum:
 
     errorMemoryAllocation = 2
 
+    cudaDevAttrMaxThreadsPerBlock             = 1  # Maximum number of threads per block
```

Comment:
Could you remove the comment part? (our coding rule (PEP8) requires 80 chars for each line)